retrieval_service/app/routes.py

```diff
@@ -148,6 +148,32 @@ async def search_flights(
     return flights
 
 
+@routes.get("/seats/search")
+async def search_seats(
```

Comment:
Sorry I missed this earlier, but we'll also have to update app_test.py for this endpoint.

Reply:
Done!